src/interface/QuotingService-swagger.yaml

```diff
@@ -1149,7 +1149,7 @@ components:
       type: object
       properties:
         extension:
-          maxItems: 16
+          maxItems: 17
```

Comment:
@shashi165 - why this change to 17?

Reply:
@elnyry-sam-k , Mowali has 17 elements, we need to check why they have 1 more (I am not making the change myself. I am just trying to merge whatever is there, I might not be able to answer why it was changed as it was changed long time ago)

Reply:
@shashi165, someone will need to log a Change Request to the FSPIOP Change Control Board for this change with the appropriate rationale.